src/UI/SignTransactionsModals/SignWithDeviceModal/components/components/ConfirmAmount/components/ConfirmAmountLabel/ConfirmAmountLabel.tsx

```diff
@@ -35,7 +38,11 @@ const ConfirmAmountLabelComponent = ({
     return (
       <div className={styles?.confirmAmountLabel}>
         <span className={styles?.confirmAmountLabelText}>You are sending</span>
-        <span className={styles?.confirmAmountLabelValue}>
+        <span
+          className={styles?.confirmAmountLabelValue}
+          data-testid={DataTestIdsEnum.confirmAmount}
+          data-value={`${amountToLocaleString} ${identifier}`}
```

Comment:
you could make a variable above
and use it again at line 46

Reply:
Updated. 👍